Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/MainActivity.cs

```diff
@@ -6,15 +6,22 @@
 using Android.Content;
 using Android.Content.PM;
 using Android.OS;
+using Contoso.Forms.Demo.Droid;
 using Microsoft.AppCenter.Push;
+using Xamarin.Forms;
 
+[assembly: Dependency(typeof(MainActivity))]
 namespace Contoso.Forms.Demo.Droid
 {
     [Activity(Label = "ACFDemo", Icon = "@drawable/icon", Theme = "@style/MyTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
-    public class MainActivity : Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    public class MainActivity : Xamarin.Forms.Platform.Android.FormsAppCompatActivity, IClearCrashClick
     {
         public static readonly int FileAttachmentId = 1;
 
+        Context mContext = Android.App.Application.Context;
```

Comment:
On Xamarin the convention is different for naming variables it would have been _context.

Here you use it only once anyway so you can remove the variable.